frontend/src/views/instructors/editable-instructors-list.js

```diff
@@ -65,7 +87,34 @@ function EditableInstructorsList(props) {
         },
     ];
 
-    return <InstructorsList columns={columns} {...rest} />;
+    function deleteButtonOnClick(instructor) {
+        setInstructorToDelete(instructor);
+        setDeleteDialogVisible(true);
+    }
+
+    const deleteColumn = {
+        // todo: weird top padding
+        Header: "Delete",
+        Cell: (props) => {
+            const instructor = props.original;
+            const disabled = workingInstructors.has(instructor.id);
+            return disabled ? null : (
+                <Button
+                    onClick={() => deleteButtonOnClick(instructor)}
+                    variant="outline-danger"
+                >
+                    <FaTrash />
```

Comment:
We already used a `FaTrash` in the side panel. Let's use an `X` icon here so the user doesn't get overwhelmed.